pkg/acsengine/engine.go

```diff
@@ -1522,6 +1614,193 @@ func getKubernetesPodStartIndex(properties *api.Properties) int {
 	return nodeCount + 1
 }
 
+// getLinkedTemplatesForExtensions returns the
+// Microsoft.Resources/deployments for each extension
+//func getLinkedTemplatesForExtensions(properties api.Properties) string {
+func getLinkedTemplatesForExtensions(properties *api.Properties) string {
+	var result string
+
+	var extensions = properties.ExtensionsProfile
+
+	//This is temporary - to show you how to access the Extensions in the MasterProfile
+	var masterProfileExtensions = properties.MasterProfile.Extensions
```

Comment:
`masterProfileExtensions := properties.MasterProfile.Extensions` is more idiomatic